backend/app/models/settings.py

```diff
@@ -5,7 +5,7 @@
 
 from app.models.types import AntiRecommenderType, RecordType
 
-CONFIG_FILE_PATH = Path(__file__).parent.parent.parent
```

Comment:
This was fine, just add .absolute() to the end of it.

Reply:
I have added .absolute() to the Paths in settings.